src/lib/UtapiClient.js

```diff
@@ -26,8 +25,8 @@ const methods = {
     multiObjectDelete: '_genericPushMetricDeleteObject',
     getObject: '_pushMetricGetObject',
     getObjectAcl: '_genericPushMetric',
-    putObject: '_pushMetricPutObject',
-    copyObject: '_pushMetricCopyObject',
+    putObject: '_genericPushMetricPutObject',
+    copyObject: '_genericPushMetricPutObject',
```

Comment:
We added a generic method for  "putObject" style requests since copyObject differs by only not having to increment `incomingBytes`.